include/ntcore_c.h

```diff
@@ -363,6 +363,11 @@ void NT_StartServer(const char* persist_filename, const char* listen_address,
 void NT_StopServer(void);
 
 /** Starts Client
+ * Starts a client.  Use NT_SetServer to set the server name and port.
+ */
+void NT_StartClientNone(void);
```

Comment:
This really isn't a test mode.  It's just allowing separation of starting up the client and setting the server addresses.